### PR TITLE
Specialize both bias and filter quantization parameters for CWQConv2D

### PR DESCRIPTION
**Pull request by mciprian13**
**Summary**
There are corner cases most commonly found in depthwise convolutions in MobileNet networks where the filter data is much smaller than bias data. For per-channel quantization, if bias quantization is not properly handled this results in bias saturation.
This PR adds logic to handle this special case by specializing both the weights and bias quantization parameters for such numerically ill-defined cases. When using INT32 bias this always ensures that biasScale = inputScale * filterScale which is implementation friendly.
Added also logic to treat the special case when bias is 0.

**Test Plan**
Added a new unit tests.

### CODE REVIEW

**Comment by mciprian13** [comment]
ping!

**Review by 842974287 — APPROVED** [review]
LGTM!

File: lib/Quantization/Base/Base.cpp [hunk 1]
```diff
@@ -566,28 +566,63 @@ TensorQuantizationParams
...
+  // only, we change the weightsScale to enforce the equality.
+  float inputScale = inputTQP.scale;
+  float weightsScale = weightsTQP.scale;
+  float biasScale = biasTQP.scale;
+  if (inputScale * weightsScale >= biasScale || biasZero) {
+    biasScale = inputScale * weightsScale;
+  } else {
+    if (biasQTy == ElemKind::Int32QTy) {
+      weightsScale = biasScale / inputScale;
+      biasScale = inputScale * weightsScale;
```
[the remark this answers is not in the document]
Comment (mciprian13) [reply lib/Quantization/Base/Base.cpp:616]: @842974287 In float32 arithmetic when computing `a = b / c` this does not mean that afterwards `b == a * c` (bit-exactly) because float32 division looses some precision. Therefore the second statement is forcing explicitly that `biasScale == inputScale * weightsScale` (bit-exactly). This equality is required in order to guarantee that at runtime for int32 bias we use shifts of 0 and a scale of 1 (which is easier to compute).

**Comment by mciprian13** [comment]
@842974287 I added the 2 unit tests also to NNPIOperatorTest blacklist.
Regarding your question: it is definitely NOT a noob question but a mathematical subtlety. The logic I use here is required for corner cases when the weights for some channels are extremely small and therefore have a very small `filterScale` parameter (extremely common in MobileNet networks). In that case the weights do not matter in the computation (since are very small) but if we would naively choose the `biasScale` as the product of `inputScale * filterScale` this would mean that when we quantize the bias we effectively saturate it resulting in large output errors (by choosing a very small `biasScale` parameter).
The unit test I added `ChannelwiseQuantizedConv2D_Int32Bias_SmallFilterData` actually proves that since without this adjustment logic the errors are much higher.

So can we have this landed? Thanks!
